extensions/wikia/PortableInfobox/tests/PortableInfoboxDataServiceTest.php

```diff
@@ -53,23 +55,72 @@ public function testSave() {
 			->save( $infoboxNode )
 			->getData();
 
-		$this->assertEquals( [ [ 'data' => [ [ 'type' => 'data', 'data' => [ 'label' => null, 'value' => 1 ] ] ],
-								 'sources' => [ 'test', 'test2' ] ] ], $result );
+		$this->assertEquals(
+			[
+				[
+					'parser_tag_version' => PortableInfoboxParserTagController::PARSER_TAG_VERSION,
+					'data' => [ [ 'type' => 'data', 'data' => [ 'label' => null, 'value' => 1 ] ] ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "label" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)